### DISCUSSION TITLE
How do we actually sync from local to server to other local device?

### DISCUSSION THREAD

**Discussion by jericopulvera** [evoluhq/evolu#303] [Q&A] [answered]
I wanna try the syncing process but it doesn't look like it's documented yet.

**Comment by steida** [comment]
It works out of the box. Any change that is made will be synced immediately or as soon as possible.

**Reply by jericopulvera** [reply]
I plan to host my nextjs client only frontend in cloudflare pages and the sync server on the vps. how do I point the sync to the vps server?

**Reply by steida** [reply, marked as answer]
https://github.com/evoluhq/evolu/blob/827c33ac212fdaa58e1f0f55b06aaa5e20090728/apps/web/components/NextJsExample.tsx#L73
